src/Error/ExceptionTrap.php

```diff
@@ -230,6 +230,7 @@ public function handleException(Throwable $exception): void
             $this->logException($exception, $request);
         }
 
+        $this->dispatchEvent('Exception.beforeRender', ['exception' => $exception]);
```

Comment:
What would be use case for this event?

Reply:
Not sure. Error.beforeRender was added as a replacement for awkward hooks so debug_kit could have access. Wonder if this one was added just to match it.

Reply:
Yes this was added so that Error and Exception subsystems had consistent paths to extend and hook into.